### ISSUE TITLE
Czech translation in progress... Almost done

### ISSUE THREAD

**Issue by andrewz1986** [valinet/ExplorerPatcher-L10N#83]
Is there a possibility to add Czech language to ExplorerPatcher? [I've finished it two years ago](https://github.com/andrewz1986/ExplorerPatcher/pull/1), but I was told new translations were not supported anymore. Now it seems that ExplorerPatcher handles translations differently, so I wanted to ask again. I wish somebody from the development team responded. I'm ready to make updates. Thanks.

**Comment by Amrsatrio** [comment]
Yes it is very possible. Just copy the strings file for English US and edit the strings to be in your language. Rename the files to ep_gui.cs-CZ.rc and ep_setup.cs-CZ.rc (if that's your language code, please confirm again), drop them into the folder with other language files, and make a PR. I'll review when I'm not packed with other stuff.

**Comment by Panzimy** [comment]
Because andrewz don't send anything i'm making a translate for Czech

**Comment by Panzimy** [comment]
And yeah I confrim that czech is code language

**Comment by andrewz1986** [comment]
Primarily, I was looking for some automated way of using my previous translations in current language system. Since it turned out everything has to be done manually, I postponed the task to later date. But I still had it on my schedule. If you would like to cooperate, Panzimy, I'm open to conversation, just send me a PM. Thanks.

**Comment by Panzimy** [comment]
Ok, how I can send you PM

**Comment by Panzimy** [comment]
And now I'm making the translation

**Comment by andrewz1986** [comment]
Sorry, I forgot that Github does not provide any PM funcionality (really?!). Use the contact you prefer:

andrewz1986@gmail.com
https://www.facebook.com/andrewz1986/

The best solution would probably be to collaborate via Google Docs:

https://docs.google.com/document/d/1rtjtPIVfzZlmnR3nl8PSykodmK24CndyEaM7WbTLI_k/edit?usp=sharing

**Comment by Panzimy** [comment]
Ok, I'm ready I send you a e-mail

**Comment by andrewz1986** [comment]
Since Panzimy seemed a little more confused than me, I sent the pull request myself. I hope I did a better job :-.) https://github.com/andrewz1986/ExplorerPatcher-L10N/pull/1/commits/d195af6e83ff028bda941a1155c6877b1566f41a

**Comment by andrewz1986** [comment]
I will be happy to cooperate with you later on testing and updating the translation, Panzimy. Since I cannot compile the code myself, we will have to wait for developers to make the next step.

**Comment by Panzimy** [comment]
Ok

**Comment by Panzimy** [comment]
Ok

**Comment by Panzimy** [comment]
@Amrsatrio We are waiting for your revision
[ep_gui.cs-CZ.txt](https://github.com/user-attachments/files/17691323/ep_gui.cs-CZ.txt)
[ep_setup.cs-CZ.txt](https://github.com/user-attachments/files/17691324/ep_setup.cs-CZ.txt)
You need to only download this file and rename it to .rc

**Comment by Amrsatrio** [comment]
https://github.com/valinet/ExplorerPatcher-L10N/pull/92
